src/SFA.DAS.Charities.Import/Startup.cs

```diff
@@ -11,6 +11,8 @@ internal class Startup : FunctionsStartup
     {
         public override void Configure(IFunctionsHostBuilder builder)
         {
+            builder.Services.AddHttpClient();
```

Comment:
Comment: Default HandlerLifeTime used. Are you sure 2 minutes is enough for your resource pool?

Reply:
Configured to be 5